docs/getting-started/install-scylla/oss-install-aws.rst

```diff
@@ -0,0 +1,112 @@
+==========================
+Launch ScyllaDB on AWS
+==========================
+
+This article will guide you through self-managed ScyllaDB deployment on AWS. For a fully-managed deployment of ScyllaDB 
+as-a-service, see `ScyllaDB Cloud documentation <https://cloud.docs.scylladb.com/>`_.
+
+To deploy ScyllaDB, you can:
+
+* Launch instances from ScyllaDB AMI (GA)
+* Launch a ScyllaDB cluster using AWS CloudFormation (for evaluation only)
+
+With the right setting, ScyllaDB image can be used for production. For the ScyllaDB Image documentation, see 
+the `ScyllaDB Machine Image project <https://github.com/scylladb/scylla-machine-image>`_.
+
+
+Launching Instances from ScyllaDB AMI 
+---------------------------------------
+
+#. Go to `Amazon EC2 AMIs – ScyllaDB <https://www.scylladb.com/download/?platform=aws#open-source>`_ in ScyllaDB's download center, 
+   choose your region, and click the **Node** link to open the EC2 instance creation wizard.
+#. Choose the instance type. See :ref:`Cloud Instance Recommendations for GCP <system-requirements-aws>` for the list of recommended instances.
+   
+   Other instance types will work, but with lesser performance. If you choose an instance type other than the recommended ones, make sure to run the :ref:`scylla_setup <system-configuration-scripts>` script.
+
+#. Configure your instance details. 
+
+   * **Number of instances** – If you are launching more than one instance, make sure to set the seed’s IPs correctly, either in the User Data (below) or after the launch.
```

Comment:
> make sure to set the seed’s IPs correctly

Is it still valid? Aren't we seedless now?

Reply:
Its still valid. Seeds are still used to find the cluster on init.
There are not part of gossip anymore.